README.md

```diff
@@ -194,19 +196,19 @@ var options = new CallOptions(headers: metadata);
 client.Alter(op, options)
 ```
 
-### Connecting To Slash Endpoint
+### Connecting To Dgraph Cloud Endpoint
 
-Please use the following snippet to connect to a Slash GraphQL or Slash Enterprise backend.
+Please use the following snippet to connect to a Slash GraphQL or Dgraph Cloud backend.
 
 
 ```c#
-var client = new DgraphClient(SlashChannel.Create("127.0.0.1:9080", "api-key-here"));
+var client = new DgraphClient(SlashChannel.Create("frozen-mango.eu-central-1.aws.cloud.dgraph.io:443", "<api-key>"));
```

Comment:
Why not use `https://` instead of `:443`?

Reply:
If you say so 👍

Reply:
Actually, I don’t really see gRPC connection strings specified with https:// , actually. It’s always host:port. https://grpc.io/docs/guides/auth. So, I'll stick with `:443` here.